modules/manage_tasks.js

```diff
@@ -0,0 +1,54 @@
+class ManageTasks {
+  constructor() {
+    this.taskArr = this.loadTasksFromLocalStorage();
+  }
+
+  addTask(taskDescrption) {
+    const newTask = {
+      description: taskDescrption,
+      completed: false,
+      index: this.taskArr.length + 1,
+    };
+
+    this.taskArr.push(newTask);
+    this.storeTasksInLocalStorage();
+  }
+
+  removeTask(index) {
+    this.taskArr.splice(index, 1);
+
+    // Update remaining properties' indices
+    this.taskArr.forEach((task, i) => {
+      task.index = i + 1;
+    });
+    this.storeTasksInLocalStorage();
+  }
+
+  editTask(index, newDescription) {
+    this.taskArr[index].description = newDescription;
+    this.storeTasksInLocalStorage();
+  }
```

Comment:
- [ ] @mutugi-c kindly check on your implementation of the editing functionality, am getting the error below whenever I try to update a task 
![image](https://user-images.githubusercontent.com/61408860/236296642-da604bbe-705a-4826-93dd-7a23d97bd2bb.png)